tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -133,7 +133,7 @@ class DuckDBPyRelation:
     def std(self, std_aggr: str, group_expr: str = ...) -> DuckDBPyRelation: ...
     def sum(self, sum_aggr: str, group_expr: str = ...) -> DuckDBPyRelation: ...
     def to_arrow_table(self, batch_size: int = ...) -> pyarrow.lib.Table: ...
-    def to_df(self) -> pandas.DataFrame: ...
+    def to_df(self, date_as_datetime: bool) -> pandas.DataFrame: ...
```

Comment:
Can you check the default is correctly declared? It should be populated in the stubs pyi as well